src/styles/product-card.css

```diff
@@ -0,0 +1,107 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+
+  width: 162px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: var(--color-block-background);
+
+  font-family: Roboto, sans-serif;
+}
+
+.product-card__image {
+  width: 160px;
+  height: 134px;
+
+  margin: 0 auto 40px;
+  object-fit: contain;
+}
+
+.product-card__title {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-text-primary);
+
+  margin-bottom: 4px;
```

Comment:
Have you reset default margins? I don't see any code that do it 👀

Reply:
Yes, I connected reset css in index.html from cdn